src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -38,7 +38,7 @@ public void execute_newPerson_success() {
 
     @Test
     public void execute_duplicatePerson_throwsCommandException() {
-        Person personInList = model.getAddressBook().getPersonList().get(0);
+        Person personInList = (Person) model.getAddressBook().getPersonList().get(0);
```

Comment:
Something feels off in the implementation here... we should not need to cast to (Person)